v2.0/foreign-key.md

```diff
@@ -49,6 +49,17 @@ For example, if you have a Foreign Key constraint on columns `(A, B)` and try to
 
 However, allowing _NULL_ values in either your foreign key or referenced columns can degrade their referential integrity. To avoid this, you can use the [Not Null constraint](not-null.html) on both sets of columns when [creating your tables](create-table.html). (The Not Null constraint cannot be added to existing tables.)
 
+### Foreign Key Actions
```

Comment:
Let's put the "new in" callout in the header instead so the newness shows up in the toc:

```
### Foreign Key Actions <span class="version-tag">New in v2.0</span>
```

Reply:
Done / Updated links